test/functional/test-size-list.js

```diff
@@ -66,15 +66,59 @@ describe('SizeList parseSizeList', () => {
     expect(res.sizes_[0].size).to.equal('111vw');
   });
 
-  it('should accept different length units including percent', () => {
+  it('should accept fractional numbers', () => {
+    const res = parseSizeList(' \n 11.1vw \n ');
+    expect(res.sizes_.length).to.equal(1);
+    expect(res.sizes_[0].mediaQuery).to.equal(undefined);
+    expect(res.sizes_[0].size).to.equal('11.1vw');
+  });
+
+  it('should accept CSS functions', () => {
+    const res = parseSizeList('screen calc(111vw + 10px) \n' +
+        ', ca_1-C((50vw+20px) / 2) ');
```

Comment:
Hmm. Good question. Please file the bug - we'll see. I think we should allow them.